src/tests/fs_tests.rs

```diff
@@ -74,39 +67,9 @@ pub mod fs_tests {
 
 
 
-    pub fn persistencetest() {
```

Comment:
Is it impossible to have an explicit test for persist now? I assume that it is pretty difficult to test now, just wanted to double check.

Reply:
Yeah it is impossible without unsafe code at which point it's really not worth it